_haikus/haikupoemJS.md

```diff
@@ -0,0 +1,9 @@
+--
```

Comment:
this should be three hyphens. you're missing one. if you fix this and push the changes to this branch, the tests should pass and we can merge.